stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/overlay/SelectedMessageOverlay.kt

```diff
@@ -281,19 +286,21 @@ public fun MessageOptions(
         shape = RoundedCornerShape(ChatTheme.dimens.messageOptionsRoundedCorners),
         color = ChatTheme.colors.barsBackground,
     ) {
-        LazyColumn(modifier) {
-            items(options) { option ->
-                Spacer(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .height(0.5.dp)
-                        .background(color = ChatTheme.colors.borders)
-                )
+        Column(modifier) {
+            options.forEach { option ->
+                key(option) {
+                    Spacer(
+                        modifier = Modifier
+                            .fillMaxWidth()
```

Comment:
Just for curious, can we use [matchParentSize](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/BoxScope#(androidx.compose.ui.Modifier).matchParentSize()) instead of the `fillMaxWidth()`?

Reply:
This is a great question Jaewoong and it's something I've been thinking about yesterday.

Unfortunately [ColumnScope](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/ColumnScope) doesn't have [.mathParentSize()](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/BoxScope#(androidx.compose.ui.Modifier).matchParentSize()) or similar width related behavior, but the same effect can be achieved through the use of [Intrinsic measurements](https://developer.android.com/jetpack/compose/layouts/intrinsic-measurements). I agree with you that simply filling the available space inside the parent would be a much more organic solution that implementing using `.fillMaxWidth()` and then using hard dimensions to cut down on size.

Were you possibly thinking of wrapping the elements inside a `Box` or is there something I'm missing?

Reply:
Oops, sorry I was mis pointed the line. I wanted to point the line 98 in this class. 😄

Reply:
We could put the matchParentSize to the column (as it's within a box) and then match the parent/wrap the items?

Alternatively, what Marin suggested makes sense to me - but if it's more work and it's overkill for this (given that we'll migrate to new UX for the component) then we can just leave it as-is.

Reply:
I've tried making it work by using the `matchParentSize()` approach but unfortunately it doesn't result in the desired outcome as it aligns the content to the top and `MessageOptions` fill maximum available horizontal space.

I already have a working solution for my proposition using `IntrinsicSize` but understand that a migration is planned, so any outcome you decide is okay by me.

I would just like a clear input on whether we will:

1. Keep this PR as is and merge it
2. Add a commit using `IntrinsicSize` and review it
3. Sync up and see if we can make it work using `.matchParentSize()`
4. Some other proposition

Reply:
Let's merge as-is and then move on with the newly proposed UI/UX in the next few weeks!